deployment/cdk/opensearch-service-migration/lib/service-stacks/traffic-replayer-stack.ts

```diff
@@ -80,21 +80,37 @@ export class TrafficReplayerStack extends MigrationServiceCore {
         });
         const groupId = props.customKafkaGroupId ? props.customKafkaGroupId : `logging-group-${deployId}`
 
-        let replayerCommand = `/runJavaWithClasspath.sh org.opensearch.migrations.replay.TrafficReplayer ${osClusterEndpoint} --insecure --kafka-traffic-brokers ${brokerEndpoints} --kafka-traffic-topic logging-traffic-topic --kafka-traffic-group-id ${groupId}`
+        let command = `/runJavaWithClasspath.sh org.opensearch.migrations.replay.TrafficReplayer ${osClusterEndpoint}`
+        const extraArgsDict = parseArgsToDict(props.extraArgs)
+        command = appendArgIfNotInExtraArgs(command, extraArgsDict, "--insecure")
```

Comment:
Note: In a followup, we need to figure out how a customer might exclude this, most likely through a cluster setting in cdk.context.json

Reply:
Seems like that's the way to go - we exclude kafkaConnection in a similar manner in the code above.